styles.css

```diff
@@ -0,0 +1,149 @@
+body {
+  font-family: Arial;
+  font-weight: 400;
+  font-style: normal;
+  letter-spacing: 0em;
+  text-transform: none;
+  line-height: 1.1em;
+}
+
+title {
+  font-size: 34 px;
+  font-family: 'Montserrat', sans-serif;
+  font-weight: 200;
+  font-style: normal;
+  letter-spacing: .1em;
+  line-Height: 1.3em;
+  text-transform: uppercase;
+  color: #0c1155;
+  text-align: center;
+  white-space: pre-wrap;
+  text-decoration: none;
+}
+
+#navFormat {
+  padding: 20px 10px;
+}
+
+.sidenav {
+  font-family: 'Roboto', sans-serif;
+  height: 100%;
+  width: 0;
+  position: fixed;
+  z-index: 1;
+  top: 0;
+  left: 0;
+  background-color: white;
+  overflow-x: hidden;
+  transition: 0.5s;
+  padding-top: 20%;
+}
+
+.sidenav a {
+  padding: 8px 8px 8px 32px;
+  text-decoration: none;
+  font-size: 25px;
+  color:black;
+  display: block;
+  transition: 0.3s;
+}
+
+.sidenav a:hover {
+  color: grey;
+}
+
+.sidenav .closebtn {
+  position: right;
+  top: 0;
+  right: 25px;
+  font-size: 40px;
+  margin-left: 50px;
+}
+
+@media screen and (max-height: 450px) {
+  .sidenav {padding-top: 15px;}
+  .sidenav a {font-size: 18px;}
+}
+
```

Comment:
I am a bit curious about the media query here. Not sure if it is being used. Not a huge deal just more curious.